api/src/main/java/gov/cms/ab2d/api/controller/AdminAPI.java

```diff
@@ -92,4 +106,13 @@
     public ResponseEntity<List<PropertiesDTO>> updateProperties(@RequestBody List<PropertiesDTO> propertiesDTOs) {
         return new ResponseEntity<>(propertiesService.updateProperties(propertiesDTOs), null, HttpStatus.OK);
     }
+
+
+    @PostMapping("/coverage/clearCache")
```

Comment:
I think this should be DeleteMapping, I can't see anything being created...you can then return an HTTP status of deleted.

Reply:
I thought about that at first. 
But one is not allowed to use a `@RequestBody` with Http DELETE verb.

Reply:
Right, you would use the HTTP parameters for the variables, there's examples in BulkDataAccessAPI controller that use {variable} in the URL.